go/cmd/dolt/commands/config.go

```diff
@@ -142,32 +142,35 @@ func getOperation(dEnv *env.DoltEnv, setCfgTypes *set.StrSet, args []string, pri
 		return 1
 	}
 
-	cfgTypesSl := setCfgTypes.AsSlice()
-	for _, cfgType := range cfgTypesSl {
-		if _, ok := dEnv.Config.GetConfig(newCfgElement(cfgType)); !ok {
-			cli.PrintErrln(color.RedString("Unable to read config."))
+	var cfg config.ReadableConfig
+	switch setCfgTypes.Size() {
+	case 0:
+		cfg = dEnv.Config
+	case 1:
+		configElement := newCfgElement(setCfgTypes.AsSlice()[0])
+		var ok bool
+		cfg, ok = dEnv.Config.GetConfig(configElement)
+		if !ok {
+			cli.Println(color.RedString("No config found for %s", configElement.String()))
 			return 1
 		}
+	default:
+		// should be impossible due to earlier checks
```

Comment:
is there any point to `StrSet` if we're not allowed to pass multiple? if `[]{'local', 'global'}` if being pre handled earlier just pass the string right?

Reply:
No real point, but it's there for symmetry with the other methods (needed by list). Didn't want to mess with it any more